src/components/views/avatars/MemberAvatar.tsx

```diff
@@ -99,3 +99,11 @@ export default function MemberAvatar({
         />
     );
 }
+
+export class LegacyMemberAvatar extends React.Component<IProps> {
```

Comment:
Can we not just use `forwardRef((props, ref) => {... <div ref={ref} />});`

Reply:
That is fair, could we open an issue to refactor NodeAnimator to be FC-friendly?